packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -136,7 +156,7 @@ const ScrollAppWrapper = ({ url, children }: ScrollAppWrapperProps) => {
 
 const BodyNormal = ({ url, menu, appMenu, children, guideOpen, isMenuInline }: NormalBodyProps) => (
     <Body>
-        <Columns>
+        <Columns guideOpen={guideOpen}>
```

Comment:
those issues with props passed to DOM elements could be solved by this new feature: https://styled-components.com/docs/api#transient-props or good old manual `export const Columns = styled(({ guideOpen, ...rest }) => <div {...rest} />)`

Reply:
I dont get any error like that. I think that the problem in the commit you posted is because it uses `animated` lib and probably something is handled wrongly there.

Reply:
ah, ok, cool